ios/INVRegionMonitor.m

```diff
@@ -138,13 +138,19 @@ - (BOOL)_isRegionMonitoringPossible:(CLAuthorizationStatus)status {
     );
 }
 
-- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit {
+- (void)_sendRegionChangeEventWithIdentifier:(NSString *)identifier didEnter:(BOOL)didEnter didExit:(BOOL)didExit locationManager:(CLLocationManager *)locationManager {
     NSDictionary *body = @{
         @"region": @{
             @"identifier": identifier,
         },
         @"didEnter": @(didEnter),
         @"didExit": @(didExit),
+        @"accuracy": @(locationManager.location != nil ? locationManager.location.horizontalAccuracy : 99999),
+        @"bearing": @(locationManager.location != nil ? locationManager.location.course : 0),
```

Comment:
Same as the comment about the location, what if location is nil? Course 0 means 0 degrees which is north. Seems to be invalid if we pass 0 if the location is nil.

Perhaps we can put the location properties, like accuracy and course or bearing, in the location property.